rules-tests/Naming/Rector/VarLikeIdentifier/SnakeToCamelCasePropertiesAndVariablesRector/Fixture/keep_constant.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+class SomeClass
+{
+    private const TEST_CONSTANT = 'test';
+}
+
+?>
+-----
+<?php
+
+class SomeClass
+{
+    private const TEST_CONSTANT = 'test';
+}
+
+?>
```

Comment:
after part is not needed for skip fixture

```suggestion
```

Reply:
Thanks :)